blueocean-core-js/.eslintrc

```diff
@@ -5,6 +5,7 @@
     "react/jsx-indent-props": 0,
     "no-unused-vars": [2, {"varsIgnorePattern": "^React$"}],
     "max-len": [1, 160, 4],
+    "prefer-template": 1,
```

Comment:
Turns off a bullshit lint rule (which is already set to 1 in the root .eslintrc) which conflicts with a different lint rule that's buggy. Using templates inside of JSX code makes the stupid single-quote-only rule shit itself, as well as being harder to read.